core/trino-main/src/test/java/io/trino/exchange/TestExchangeConfig.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.exchange;
+
+import com.google.common.collect.ImmutableMap;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Map;
+
+import static io.airlift.configuration.testing.ConfigAssertions.assertFullMapping;
+import static io.airlift.configuration.testing.ConfigAssertions.assertRecordedDefaults;
+import static io.airlift.configuration.testing.ConfigAssertions.recordDefaults;
+
+public class TestExchangeConfig
+{
+    @Test
+    public void testDefaults()
+    {
+        assertRecordedDefaults(recordDefaults(ExchangeManagerConfig.class)
+                .setExchangeManagerConfigFile(null));
+    }
+
+    @Test
+    public void testExplicitPropertyMappings()
```

Comment:
```suggestion
    void testDefaults()
    {
        assertRecordedDefaults(recordDefaults(ExchangeManagerConfig.class)
                .setExchangeManagerConfigFile(null));
    }

    @Test
    void testExplicitPropertyMappings()
```

https://trino.io/docs/current/develop/tests.html#conventions-and-recommendations
* Test methods should be defined as package-private.

Reply:
done